modules/ROOT/pages/create-outbound-message-flow.adoc

```diff
@@ -64,6 +64,8 @@ To import the translation map:
 . Navigate to the DataWeave map you want to use and click *Open*.
 . Click *Upload mapping*.
 
+After you import the translation map for an outbound message flow, you can access source message attributes in that map. This makes these attributes available in the message playoad. The attributes that you can access depend on the protocol of the source endpoint. For more information, see xref:access-source-message-attributes.adoc[Accessing Source Message Attributes in Translation Maps]. 
```

Comment:
Please change "After you import the translation map for an inbound message flow.....", "In the translation map you develop for an outbound message flow..."